rubocop.yml

```diff
@@ -242,3 +248,45 @@ Style/HashLikeCase:
 
 Style/RedundantFileExtensionInRequire:
   Enabled: true
+
+Lint/BinaryOperatorWithIdenticalOperands:
+  Enabled: true
+
+Lint/DuplicateRescueException:
+  Enabled: true
+
+Lint/EmptyConditionalBody:
+  Enabled: true
+
+Lint/FloatComparison:
+  Enabled: true
+
+Lint/MissingSuper:
+  Enabled: true
+
+Lint/OutOfRangeRegexpRef:
+  Enabled: true
+
+Lint/SelfAssignment:
+  Enabled: true
+
+Lint/TopLevelReturnWithArgument:
+  Enabled: true
+
+Lint/UnreachableLoop:
+  Enabled: true
+
+Style/ExplicitBlockArgument:
+  Enabled: true
+
+Style/GlobalStdStream:
+  Enabled: true
+
+Style/SingleArgumentDig:
+  Enabled: true
+
+Style/OptionalBooleanParameter:
+  Enabled: false
```

Comment:
Not sure if enabling this would work with que?

Reply:
Yeah this would probably conflict with our Que Jobs, wonder if we can `exclude` the jobs that use this in our services?

Reply:
Going to keep it disabled for now, we can't come back and enable it latert as there's already a new rubocop release out of the door